sdk/armada-protocol-common/src/common/interfaces/IArmadaRebalanceData.ts

```diff
@@ -27,26 +27,28 @@ export interface IRebalanceData {
 }
 
 /**
- * @description Zod schema for IRebalanceData
+ * @description Zod schema for IArmadaRebalanceData
  */
-export const RebalanceDataSchema = z.object({
+export const ArmadaRebalanceDataSchema = z.object({
   fromArk: AddressDataSchema,
   toArk: AddressDataSchema,
   amount: TokenAmountDataSchema,
 })
```

Comment:
```suggestion
  boardData: z.string(),
  disembarkData: z.string(),
})
```

Reply:
Thanks! I've improved the data type by using `HexData` and checking with `isHex` similar to what is done in `IAddress`